src/components/VariantDb/index.tsx

```diff
@@ -229,7 +229,7 @@ class VariantDb extends React.Component<VariantDbProps, VariantDbState> {
                         color: '#7D84A6',
                       }}
                     >
-                      May 13, 2020
+                      May 21  , 2021
```

Comment:
there few extra spaces before the coma
shouldn't be 'January 21, 2021' instead ?